### DISCUSSION TITLE
VLAN 4093

### DISCUSSION THREAD

**Discussion by JHAvB** [fabianishere/udm-iptv#5] [Q&A]
Thanks again for this page.

SSHnet (NL) uses VLAN 66 for internet and VLAN 4093 for IPTV. For internal network it is not possible to go higher than VLAN 4018. 

Does this also apply to WAN? I am not getting an IP on VLAN 4093.

**Comment by fabianishere** [comment]
The limits from Linux apply, so for the WAN side any VLAN 1-4094. It guess the DHCP settings are not correct.
